metis-core/metis-core-rest/src/main/resources/application.properties.example

```diff
@@ -1,11 +1,17 @@
 #Spring
```

Comment:
suggestion to add a comment, inside pointing to use the metis-configuration approach.

Reply:
I would prefer not to do that, since then we'd have to repeat that on other files.